Examples/Scotty-tvOSExample/Routing/RouteController+App.swift

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
Similar comment to above about extracting this into a shared `Examples/Common` folder and then making it a member of both tvOS and iOS targets.